src/submission/submission-error-messages.ts

```diff
@@ -55,6 +55,14 @@ const ERROR_MESSAGES: { [key: string]: (errorData: any) => string } = {
   TREATMENT_ID_NOT_FOUND: () => {
     return `Treatment and treatment_type files are required to be initialized together. Please upload a corresponding treatment file in this submission.`;
   },
+  MISSING_VARIABLE_REQUIREMENT: errorData => {
+    const info = errorData.info;
+    return `${errorData.fieldName} is a required field because the corresponding record in ${info.referenceSchema}.tsv had ${info.variableRequirement.fieldName} set to ${info.variableRequirement.fieldValue}.`;
```

Comment:
can you please confirm the error messages with Kim if not already

Reply:
okay, will ask her